### ISSUE TITLE
"Experience \"@[secure]/expomultiscreenstarter\" for platform \"[secure]\" and release channel \"default\" does not exist or is not published.

### ISSUE THREAD

**Issue by liudonghua123** [expo/turtle#144]
### New Issue Checklist

- [x] Updated turtle-cli to the latest version (`yarn global add turtle-cli` or `npm install --global turtle-cli`).
- [x] I read the [Contribution Guidelines](https://github.com/expo/turtle/blob/master/CONTRIBUTING.md).
- [x] I read [docs.expo.io](https://docs.expo.io) (Managed Workflow > Distributing Your App).
- [x] I searched for [existing GitHub issues](https://github.com/expo/turtle/issues).
- [x] **I enclosed the complete output when running turtle, including the stack trace and command used.**

### Issue Description

I write a travis ci config file according https://github.com/expo/turtle-cli-example/blob/master/.travis.yml, but when the ci execute `turtle build:android --keystore-path ./expo-project.jks --keystore-alias $EXPO_ANDROID_KEYSTORE_ALIAS --type apk -o $ARTIFACT_PATH`, it always gave me `turtle[5266] ERROR: { StatusCodeError: 404 - "Experience \"@[secure]/expomultiscreenstarter\" for platform \"[secure]\" and release channel \"default\" does not exist or is not published."` error.

more detail can be found on 
https://travis-ci.org/liudonghua123/expo-multi-screen-starter/jobs/591506731

I can confirm that environment variables are correct and `https://expo.io/@liudonghua/expomultiscreenstarter` is reachable.

##### Command executed

see the travis log above.

##### Complete output when running turtle, including the stack trace and command used

see the travis log above.

### Environment

see the travis log above.

**Comment by wkozyra95** [comment]
Did you publish your app after upgrading to sdk 35?

**Comment by liudonghua123** [comment]
@wkozyra95 Hi, I have published my app and I create my app using the latest sdk 35.

**Comment by dsokal** [comment]
Please run `expo publish` in the project dir and make sure you're logged in as the same user you're using to build your app on Travis (check that with `expo whoami`). Then, build your app with Travis again.

**Comment by liudonghua123** [comment]
@dsokal Hi, I tried your suggestion, but the same error log stills. I executed `expo publish` successfully, and I can confirm that the publish url `https://exp.host/@liudonghua/expomultiscreenstarter` was generated.

![image](https://user-images.githubusercontent.com/2276718/67614911-77c6a280-f7f7-11e9-8236-2a837d375784.png)

**Comment by dsokal** [comment]
I'm 99.99% sure you're logging in as a different user on Travis. Why do I think so?

I tried to fetch the manifest for `@liudonghua/expomultiscreenstarter` and it suceeded:
```bash
 dsokal@Dominiks-CebulaBook  ~  curl -X GET \
  https://exp.host:443/@liudonghua/expomultiscreenstarter \
  -H 'Accept: application/expo+json,application/json' \
  -H 'Expo-Release-Channel: default' \
  -H 'Exponent-Platform: android' \
  -H 'Exponent-SDK-Version: 35.0.0' \
  -H 'Host: exp.host:443'
{"android":{"versionCode":2},"bundledAssets":["asset_6b6170fe52fb23f505b4e056fefd2679.ttf","asset_2aca33bf4f4ab838b6ae6f7750bf00c0.png","asset_0f3763e16e82cb3de89fef9ca7f20770.png"],"dependencies":["expo","expo-asset","expo-font","expo-image-picker","prop-types","react","react-dom","react-native","react-native-appearance","react-native-elements","react-native-gesture-handler","react-native-reanimated","react-native-svg","react-native-web","react-navigation","react-navigation-stack","react-navigation-tabs"],"description":"react native multi screen application starter with expo","githubUrl":"https://github.com/calebnance/expo-multi-screen-starter","icon":"src/assets/icon.png","iconUrl":"https://d1wp6m56sqw74a.cloudfront.net/~assets/bf80a4df91e6339614a4813155907677","ios":{"buildNumber":"0.0.2","supportsTablet":true},"locales":{},"name":"Multi-screen Starter","orientation":"portrait","platforms":["android","ios","web"],"primaryColor":"#2a2a2a","privacy":"public","scheme":"expomultiscreenstarter","sdkVersion":"35.0.0","slug":"expomultiscreenstarter","splash":{"backgroundColor":"#2a2a2a","image":"src/assets/splash.png","imageUrl":"https://d1wp6m56sqw74a.cloudfront.net/~assets/b58678d99d71f759d1040d1cd5f2c645","resizeMode":"cover"},"updates":{"fallbackToCacheTimeout":0},"version":"0.0.2","id":"@liudonghua/expomultiscreenstarter","revisionId":"0.0.2-r.SJNEnPbqS","publishedTime":"2019-10-26T06:48:12.388Z","commitTime":"2019-10-26T06:48:13.014Z","bundleUrl":"https://d1wp6m56sqw74a.cloudfront.net/%40liudonghua%2Fexpomultiscreenstarter%2F0.0.2%2Fe8c2b42635ae0e6554054e82aa58f99a-35.0.0-android.js","releaseChannel":"default","hostUri":"exp.host:443/@liudonghua/expomultiscreenstarter"}%                                                                      
```

Please note the `githubUrl ` key:
`"githubUrl": "https://github.com/calebnance/expo-multi-screen-starter",`

I think you're logging ing as `calebnance` on Travis and you should log in as `liudonghua`.

I tried to fetch the manifest for your other user and it failed with the same error as you're getting on Travis:
```bash
dsokal@Dominiks-CebulaBook  ~  curl -X GET \                                                                                      
  https://exp.host:443/@calebnance/expomultiscreenstarter \
  -H 'Accept: application/expo+json,application/json' \
  -H 'Expo-Release-Channel: default' \
  -H 'Exponent-Platform: android' \
  -H 'Exponent-SDK-Version: 35.0.0' \
  -H 'Host: exp.host:443'
Experience "@calebnance/expomultiscreenstarter" for platform "android" and release channel "default" was not published recently enough to be used on this client. The author will need to publish a new version.%    
```

Please check the env vars on Travis and correct me if I'm wrong.

**Comment by liudonghua123** [comment]
@dsokal Hi, thanks for these helpful tips, The repo was clone from https://github.com/liudonghua123/expo-multi-screen-starter. I tried to search `calebnance` in the source code, and I found these is one match `"githubUrl": "https://github.com/calebnance/expo-multi-screen-starter"` in `/app.json`, and I replaced with `"githubUrl": "https://github.com/liudonghua123/expo-multi-screen-starter"`. and I also checked the environments on Travis, the `EXPO_USERNAME` is `liudonghua` and you can see the log `expo login` was successfully. But when I tried another build on Travis, the same errors like before.
My github username is `liudonghua123` and my expo username is `liudonghua`, does it affect?

**Comment by liudonghua123** [comment]
@dsokal Finally, I made it worked. First I filled my github username info in expo profile.
![image](https://user-images.githubusercontent.com/2276718/67728271-9c08c600-fa27-11e9-9930-658f1b196a05.png)

Then I tried to generate and upload the keystore use `expo credentials:manager` and get detail info use `expo fetch:android:keystore`
![image](https://user-images.githubusercontent.com/2276718/67728466-4b459d00-fa28-11e9-8a9f-3a146519f1ec.png)

And I also tried to modify `app.json` to include `android.package` field.

Thanks very much.
